spec/models/user_account_spec.rb

```diff
@@ -41,6 +41,60 @@
     end
   end
 
+  describe 'scopes' do
+    describe '#idme_verification' do
+      subject { described_class.idme_verification }
+
+      context 'when there are accounts with idme UserVerifications' do
+        let(:idme_user_verification) { create(:idme_user_verification) }
+        let!(:user_account) { idme_user_verification.user_account }
+
+        it 'returns those accounts' do
+          expect(subject.count).to eq(1)
```

Comment:
Newbie question 🙇‍♀️ : Is it preferred here to test scopes via a count rather than demonstrating the specific expected records are returned? (Previous jobs have preferred testing for the records, so I want to be sure what the standards are here! Thanks!)

Reply:
Probably depends case-by-case. In this case I don't think it really matters since the objects are contained, but probably best to just check it anyway.

Reply:
Additionally there are opinions out there that scopes shouldn't even be tested directly. These were just a bit more complex so I threw some in.

Reply:
Appreciate the clarification! Every place seems to have their own opinions on how to test scopes so I figured it wouldn't hurt to ask. Thank you!